extensions/npm/main.ts

```diff
@@ -94,19 +103,33 @@ if (Deno.args.length >= 1
         console.log(`[${green("phylum")}] Packages built successfully.`);
     }
 } else {
-    let status = await Deno.run({ cmd: ['npm', ...Deno.args] }).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'npm', 
+        args: Deno.args,
+        exceptions: {
+            write: ['~/.npm', './'],
+            read: ['~/.npm', './'],
+            run: ['npm'],
+            net: true,
+        }
+    });
     Deno.exit(status.code);
 }
 
 // Analyze new packages.
 async function checkDryRun(subcommand: string, args: string[]) {
     console.log(`[${green("phylum")}] Updating lockfile…`);
 
-    let status = await Deno.run({
-        cmd: ['npm', subcommand, '--package-lock-only', ...args],
-        stdout: 'piped',
-        stderr: 'piped',
-    }).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'npm',
+        args: [subcommand, '--package-lock-only', ...args],
+        exceptions: {
+            write: ['~/.npm', './'],
+            read: ['~/.npm', './'],
+            run: ['npm'],
+            net: true,
+        }
+    })
```

Comment:
Have you tested this with local packages? It should also need `read: true`, right?

Reply:
I haven't but you're right, `read: true` should be good since we're not executing scripts anyway.